api/controllers/userController.js

```diff
@@ -0,0 +1,38 @@
+const {userService} = require("../services");
+const {catchAsync} = require("../utils/error");
+
+const kakaoSignIn = catchAsync(async (req, res) => {
+    const {code} = req.query;
+    if (!code) {
+        res.status(200).json({message: "KEY_ERROR"});
+    }
+    const result = await userService.login(code);
+    res.status(200).json({result});
```

Comment:
```suggestion
    res.status(200).json({result});
```
service 함수에서 status code 분기에 따라 다르게 return하고 있는데 사용을 안하고 계시네요! controller가 수정되거나 service 로직이 수정되거나 해야할 것 같습니다.